packages/client-sdk-nodejs/src/batchutils/batch-props.ts

```diff
@@ -0,0 +1,36 @@
+import {CacheGet, CacheSet, CacheDelete} from '@gomomento/sdk-core';
+import {CacheClient} from '../cache-client';
+
+export const defaultMaxConcurrentRequests = 5;
+
+export interface BatchGetRequest {
+  cacheClient: CacheClient;
```

Comment:
I don't see anything here that makes this node specific, why not tie this to the `ICacheClient` interface from core, and then we can implement this in both nodejs and web-sdk?